src/sampleManifest.tsx

```diff
@@ -68,6 +69,7 @@ export const sampleManifest: SampleSpecGroup[] = [{
     getTooltipCustomizeSpec(),
     getViewClipSpec(),
     getZoomToElementsSpec(),
+    getReadSettingsSpec(),
```

Comment:
Up to this point, this list has been alphabetical based on the name of the name of the samples.

Reply:
Reordered alphabetically.